lib/info.js

```diff
@@ -54,6 +55,11 @@ exports.getBasicInfo = (id, options, callback) => {
       }
     }
 
+    const captchaForm = util.between(body, '<div class="g-recaptcha"', '>');
+    if(captchaForm) {
+      return callback(Error("reCAPTCHA form shown"));
```

Comment:
use single quotes, and let's reword this to have a little more information on why the form was shown. is it shown when made too many requests, such as the 429 error? does the form say anything?

Reply:
That's the HTML code of the form:

```
<div id="content" class="  content-alignment" role="main"><p class='largeText'>Sorry for the interruption. We have been receiving a large volume of requests from your network.</p>

<p>To continue with your YouTube experience, please fill out the form below.</p>
<form action="/das_captcha" method="POST">
<div class="g-recaptcha" data-sitekey="6Lf39AMTAAAAALPbLZdcrWDa8Ygmgk_fmGmrlRog"></div>
<br/>
<input type="hidden" name="next" value="/watch?v=s9xtPSweJNI&amp;hl=en&amp;bpctr=1574634970">
<input type="hidden" name="session_token" value="QUFFLUhqbE9talladHA2NmxZUk5CenQxZXU3WlQwSkYtUXxBQ3Jtc0ttNjB0UFgzRzU1bXRvU054VnJ2X01sMGh4Y3RKZHR0VUwtNldWZ045ZS1EbWFXbHJ1Rkh2TXYxTDJoNDljenFtOGtVbjlwZ3pTdktEZ3VkdjBFMnBBTEtGX0VqMUVUdzJ0NG13UV9ILUtzQzdvcW9IdTd6VFZTUzFfcjhJTWc2RF9BZUhXM3JSRVJENmJVMnBTSDVmaHBTUUJ6T1E=">
<input type="hidden" name="action_recaptcha_verify2" value="1">
<input type="submit" value="Submit">
</form>
```

Reply:
how about the text from the `largeText` element?

Reply:
would be cool to export `data-sitekey` and `session_token` since (as far as i know) those two are everything that's needed to solve the captcha
maybe an optional `option.captchaSolver = (siteKey, sessionToken) => { ... }` ?